packages/react/src/Timeline/Timeline.module.css

```
@@ -0,0 +1,52 @@
+.Timeline {
+  --bulletY: calc(var(--brand-Timeline-lineHeight) / 2 - var(--brand-Timeline-bullet-size) / 2 - 1px);
+
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+.Timeline__item {
+  position: relative;
+  max-width: var(--brand-Timeline-lineLength);
+  padding-inline-start: calc(var(--brand-Timeline-bullet-size) + var(--brand-Timeline-spacing));
+  color: var(--brand-color-text-default);
+  line-height: var(--brand-Timeline-lineHeight);
+}
+
+.Timeline__item:not(:last-child) {
+  padding-bottom: var(--base-size-32);
+}
+
+.Timeline__item::before,
+.Timeline__item::after {
+  content: '';
+  position: absolute;
+}
+
+.Timeline__item::before {
+  width: var(--brand-Timeline-bullet-size);
+  height: var(--brand-Timeline-bullet-size);
+  top: var(--bulletY);
+  left: 0;
+  border-style: solid;
+  border-width: var(--brand-Timeline-strokeWidth);
+  border-color: var(--brand-Timeline-line-color);
+  border-radius: 100%;
+}
+
+.Timeline__item:not(:last-child)::after {
+  top: calc(var(--bulletY) + var(--brand-Timeline-bullet-size) + var(--brand-Timeline-bullet-spacing));
+  left: calc(var(--brand-Timeline-bullet-size) / 2);
+  width: var(--brand-Timeline-strokeWidth);
+  height: calc(100% - var(--brand-Timeline-bullet-size) - var(--brand-Timeline-bullet-spacing) * 2);
+  transform: translateX(-50%);
+  background-color: var(--brand-Timeline-line-color);
+  border-radius: var(--brand-Timeline-strokeWidth);
```

Should we use a border-radius token here too?

```suggestion
  border-radius: var(var(--brand-borderRadius-full);
```

---

We can't use `100%` here, otherwise the line would have an elliptical shape.